src/status_im/ui/screens/advanced_settings/views.cljs

```diff
@@ -115,7 +117,15 @@
      #(re-frame/dispatch
        [:multiaccounts.ui/switch-wallet-connect-enabled (not wallet-connect-enabled?)])
      :accessory               :switch
-     :active                  wallet-connect-enabled?}]))
+     :active                  wallet-connect-enabled?}
+    (when (and config/quo-preview-enabled? platform/android?)
```

Comment:
why only android? i believe designers test on ios

Reply:
hm not sure i understand why toggle is os dependent?

Reply:
Toggle is working great also in ios, but enabled new UI has several issues.
1. Screens are overlapping status-bar
2. Switcher animations look weird
3. App is crashing on opening chat with nav2 dispatch
So I am thinking to remove the os flag, once these issues are addressed.

Reply:
do we reset root ? in that case it should be good , no difference with other screens